articles/active-directory/develop/quickstart-register-app.md

```diff
@@ -60,6 +60,9 @@ Your application's code, or more typically an authentication library used in you
 
 :::image type="content" source="media/quickstart-register-app/portal-03-app-reg-02.png" alt-text="Screenshot of the Azure portal in a web browser, showing an app registration's Overview pane.":::
 
+> [!NOTE]
+> There can be multiple applications with the same name but different client ID in Azure Active Directory.
+
```

Comment:
@NandanHegde15 Thanks for your contribution! Before I accept it, however, I'd like to request a change to increase clarity.

Counter to expectations, the content in alert boxes like **Note** and **Important** has been found to be _less_ visible to readers, especially in documentation sets like this one in which they're currently overused. They might see the box, but they'll skip reading its content because they consider it extra or optional information (yes, even if it says it's _Important_) or because they have "box fatigue," the result of seeing too many alert boxes too often in the documentation they've been reading.

Instead, make your content more visible by including it as regularly formatted text but _within context_. That is, add the text where the reader needs to know about it.

In this case, that means adding right there in step 5 where we ask the reader to enter the name of their application. I'd recommend going with something like this:

> 5. Enter a display **Name** for your application. Users of your application might see the display name when they use the app, for example during sign-in.
> 
>     You can change the display name at any time and multiple app registrations can share the same name. The app registration's automatically generated *Application (client) ID*, not its display name, uniquely identifies your app within the identity platform.